source/modulo_controllers/include/modulo_controllers/BaseControllerInterface.hpp

```diff
@@ -81,6 +81,7 @@ typedef std::variant<
  * @brief Input structure to save topic data in a realtime buffer and timestamps in one object.
 */
 struct ControllerInput {
+  ControllerInput() = default;
```

Comment:
Why do we need that?

Reply:
Because std::move will not compile for std::any (can't be moved) that is part of BufferVariant, so there needs to be a viable alternative. We could consider asserting false or printing an error in the constructor.